src/QirRuntime/test/Standalone/InputTypes/input-types.qs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR
+{
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation InputTypes(anInt : Int, aDouble : Double, aBool : Bool, aString : String, anArray : Int[]) : Unit {
+        
```

Comment:
Maybe, add a comment on why there is no code here? What exactly are we testing/concept-proving?

Reply:
Also, why do we need two samples: InputTypes _and_ RandomBit?

Reply:
The concept is to have a few examples to know how the generated C++ driver should look like. _RandomBit_ was just the first example that worked as a standalone executable, then _InputTypes_ is meant to look like a program that receives all the supported entry-point input types.

I also have to add an _OutputTypes_ that returns all the supported output types. I suppose all of that can be done in one example but I think that for reference is better to have two programs because it is easier to see what is going on.

Regarding why there is no code here, since this is just to prove the concept of the C++ driver, programs don't really have to do anything. Once that Robin's PR supporting `Message` is merged I'll add a few `Message` calls with the values of the input arguments to make it easy to verify that the driver is doing its job.

I'll add a readme.md to the root of Standalone root explaining this.

Reply:
Added readme.md.